wp-modules/editor/package.json

```diff
@@ -19,7 +19,7 @@
     "@wordpress/i18n": "^4.3.1",
     "@wordpress/icons": "^9.16.0",
     "@wordpress/plugins": "^5.4.0",
-    "@wordpress/scripts": "^26.8.0",
+    "@wordpress/scripts": "^26.15.0",
```

Comment:
This 1-line edit caused a 6.938-line diff for `wp-modules/editor/package-lock.json` 🤣